src/frontend/get-jira-configuration.ts

```diff
@@ -50,18 +51,26 @@ const formatDate = function (date) {
 	};
 };
 
-export const getFailedConnections = (installations, subscriptions) => {
-	return installations
+export const getFailedConnections = (
+	installations: FailedInstallations[] | typeof Installation[],
```

Comment:
This seems weird.  Fairly sure the only thing that should be coming in is `FailedInstallations[]`

Reply:
Not at this point. We pass in all installations and then filter out the ones without errors on line 59.

Reply:
I think the type should be `installations: (FailedInstallations | InstallationClass)[]` as they're mixed within the same array and not 2 separate arrays of different types.  Also, do `import InstallationClass from "../models/installation"` instead of `typeof Installation`.  This is the same issue as mentioned before with Subscription.